### ISSUE TITLE
Small bug in configuration of plugins (Athena)

### ISSUE THREAD

**Issue by roentgenium** [bruceravel/demeter#23]
I might find two small bugs in configuration of plugins of Athena.

# "Apply and Save" does not save changed parameters after restart. (eg. X15B plugin)

## Procedure

1. Change i0 from 6 (default) to 5, click "Apply and Save" and click "Cancel".
2. Exit and restart Athena.
3. The i0 in configuration of X15B plugin is 6.

# "Reset all parameters to Demeter's defaults" aborts Athena. (eg. X15B plugin)

## Procedure

1. Just click "Reset all parameters to Demeter's defaults" and "Reset", then Athena aborts with the following message.

```
Can't call method "status" on unblessed reference at C:/strawberry/perl/site/lib/Demeter/UI/Wx/Config.pm line 360.
```

**Comment by bruceravel** [comment]
Both issues are addressed in 8085a23b8496a6fbd69eedec16b61b6957575d13.  If you could pull, rebuild, and give it a test, I'd be grateful.

The first one was quite interesting and was a fairly deep bug in the design of the configuration system.  I am grateful that you found it.

The second one was easily fixed, but I decided to simply remove the "reset all" button.  The function attached to it is not to my liking.  It may reappear in the future in a different form.

Thanks a million!

**Comment by roentgenium** [comment]
I successfully rebuild the present HEAD branch (947effa) on Debian GNU/Linux 8 jessie on a virtual machine, and confirmed the issues are fixed.

Thanks again!
